packages/battery/example/test_driver/battery_e2e.dart

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:e2e/e2e.dart';
+import '../lib/main.dart';
+
+void main() {
+  E2EWidgetsFlutterBinding.ensureInitialized();
+
+  testWidgets('Builds and runs', (WidgetTester tester) async {
+    await tester.pumpWidget(MyApp());
+  });
```

Comment:
While we're here, I think it would be worthwhile to integration test the functionality a bit more, either for the example app or the plugin or both.

Reply:
done (and moved out of the example folder)